nengo_loihi/hardware/snips/nengo_host.cc.template

```diff
@@ -102,7 +110,8 @@ class NengoHostProcess : public ConcurrentHostSnip {
             }
 
             // write packets to chip (PACKET_SIZE == sizeof(int32_t))
-            writeChannel(input_channel, buffer, n_read / sizeof(int32_t));
+            size_t n_packets = ceil_div<size_t>(n_read, 4*PACKET_SIZE);
```

Comment:
The packet size in bytes `4*PACKET_SIZE` can be converted to `PACKET_BYTE_SIZE` to have better clarity on why the packet size is multiplied by 4